.pipelines/oneBranch.ACNBuild.yaml

```diff
@@ -0,0 +1,38 @@
+pr:
+  branches:
+    include:
+      - master
+trigger:
+  branches:
+    include:
+      - master
+variables:
+
+  WindowsContainerImage: onebranch.azurecr.io/windows/ltsc2019/vse2022:latest # for Windows jobs
+  LinuxContainerImage: mcr.microsoft.com/onebranch/cbl-mariner/build:2.0 # for Linux jobs
+name: '$(TeamProject)_$(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)'
+resources:
+  repositories:
+  - repository: templates
+    type: git
+    name: OneBranch.Pipelines/GovernedTemplates
+    ref: refs/heads/main
+extends:
+  template: v2/OneBranch.Official.CrossPlat.yml@templates
+  parameters:
+    stages:
+    - template: pipeline.yml
```

Comment:
Because we have so many pipeline.yml files, can you put a comment to the right for the name of display name of the pipeline `Azure Container Networking PR`